### PR TITLE
chore: Select component refactoring - TimeSeriesColumnControl - Iteration 5

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
It replaces the react-select Select component with the Antdesign one in the TimeSeriesColumnControl component.

### BEFORE

https://user-images.githubusercontent.com/60598000/130768485-f694c7ec-b6e3-43d0-b178-93a8b7326be8.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/130768576-9d61f25f-c677-490b-b16b-ce2130f441df.mp4

### TESTING INSTRUCTIONS
1. Open a Time-series Table Chart in Explore
2. Add time columns and observe the dropdown behavior

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by geido** [comment]
@rosemarie-chiu @jinghua-qa

File: superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx [hunk 1]
```diff
@@ -216,8 +216,9 @@ export default class TimeSeriesColumnControl extends React.Component {
           'Type of comparison, value difference or percentage',
           'col-type',
           <Select
-            value={this.state.colType}
-            clearable={false}
+            ariaLabel="col-type"
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx:219]:
```suggestion
            ariaLabel={t('Type')}
```

File: superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx [hunk 2]
```diff
@@ -216,8 +216,9 @@ export default class TimeSeriesColumnControl extends React.Component {
...
+            value={this.state.colType || undefined}
+            allowClear={false}
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx:221]:
```suggestion
```

File: superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx [hunk 3]
```diff
@@ -273,8 +274,9 @@ export default class TimeSeriesColumnControl extends React.Component {
             'Type of comparison, value difference or percentage',
             'comp-type',
             <Select
-              value={this.state.comparisonType}
-              clearable={false}
+              ariaLabel="comparison-type"
+              value={this.state.comparisonType || undefined}
+              allowClear={false}
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/controls/TimeSeriesColumnControl/index.jsx:279]:
```suggestion
              ariaLabel={t('Type')}
              value={this.state.comparisonType || undefined}
```

**Review by michael-s-molina — APPROVED** [review]
Code LGTM. Left some non-blocking suggestions. It would be nice to have QA approval.

**Comment by michael-s-molina** [comment]
To fix `TypeError: handleOnSearch.cancel is not a function` in your tests you have two options:
- Don't mock `debounce` and just account for the async behavior.
- Include the `cancel` function is your mock.
```
jest.mock('lodash/debounce', () => (fn: Function) => ({
  ...fn,
  cancel: jest.fn(),
}));
```

**Pull request merged**